.gitignore

```diff
@@ -1,5 +1,7 @@
 build**
 dist**
+plugins**
```

Comment:
Adding this to .gitignore is ok, but it then requires adding a `files` section to `package.json` so that the packaging will include the plugins.